test/e2e/install_test.go

```diff
@@ -180,48 +180,56 @@ var _ = Describe("Operator Install", func() {
 			}).Should(Succeed())
 		})
 
-		It("handles upgrade edges correctly", func() {
-			By("creating a valid Operator resource")
-			operator.Spec = operatorv1alpha1.OperatorSpec{
-				PackageName: "prometheus",
-				Version:     "0.37.0",
-			}
-			Expect(c.Create(ctx, operator)).To(Succeed())
-			By("eventually reporting a successful resolution")
-			Eventually(func(g Gomega) {
-				g.Expect(c.Get(ctx, types.NamespacedName{Name: operator.Name}, operator)).To(Succeed())
-				cond := apimeta.FindStatusCondition(operator.Status.Conditions, operatorv1alpha1.TypeResolved)
-				g.Expect(cond).ToNot(BeNil())
-				g.Expect(cond.Reason).To(Equal(operatorv1alpha1.ReasonSuccess))
-				g.Expect(cond.Message).To(ContainSubstring("resolved to"))
-				g.Expect(operator.Status.ResolvedBundleResource).ToNot(BeEmpty())
-			}).Should(Succeed())
+		When("resolving upgrade edges", func() {
```

Comment:
Could we add some `0.minor.patch` and `0.0.patch` version upgrades? Would be great to codify expectations.

Reply:
I ended up doing a separate PR here: https://github.com/operator-framework/operator-controller/pull/472